blueflood-elasticsearch/src/test/java/com/rackspacecloud/blueflood/io/ElasticTokensIOTest.java

```diff
@@ -1,27 +1,39 @@
 package com.rackspacecloud.blueflood.io;
 
+import com.rackspacecloud.blueflood.types.IMetric;
 import com.rackspacecloud.blueflood.types.Locator;
+import com.rackspacecloud.blueflood.types.Metric;
 import com.rackspacecloud.blueflood.types.Token;
 import com.rackspacecloud.blueflood.utils.GlobPattern;
-import org.junit.BeforeClass;
+import com.rackspacecloud.blueflood.utils.TimeValue;
+import org.junit.Before;
 import org.junit.Test;
+import org.mockito.ArgumentCaptor;
 
 import java.io.IOException;
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
+import java.util.Random;
+import java.util.concurrent.TimeUnit;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.*;
 import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.*;
 
 public class ElasticTokensIOTest {
 
-    protected static ElasticTokensIO elasticTokensIO;
+    protected ElasticTokensIO elasticTokensIO;
+    private ElasticsearchRestHelper mockElasticsearchRestHelper;
 
-    @BeforeClass
-    public static void setup() throws IOException {
+    @Before
+    public void setup() throws IOException {
         elasticTokensIO = new ElasticTokensIO();
+        mockElasticsearchRestHelper = mock(ElasticsearchRestHelper.class);
+        elasticTokensIO.setElasticsearchRestHelper(mockElasticsearchRestHelper);
```

Comment:
TokenCache.getInstance().resetCache(); may be needed to make tests happy.

Reply:
[Fixed](https://github.com/rax-maas/blueflood/compare/61529b6eb6fb894c2ff360bb3b628dd73b71edf3..c009776e134915e5417c6642419dd20ea740f894#diff-4e793beb5eafe1d535bc7b3ec8a9bd32a6745f0b51a846e91a38bf0f7401da9bR35). Tests should be happy now.